languages.toml

```
@@ -1493,7 +1493,16 @@ name = "dockerfile"
 scope = "source.dockerfile"
 injection-regex = "docker|dockerfile"
 roots = ["Dockerfile", "Containerfile"]
-file-types = [{ glob = "Dockerfile*" }, { glob = "dockerfile*" }, { glob = "Containerfile*" }, { glob = "containerfile*" }]
+file-types = [
+  { glob = "Dockerfile*" },
+  { glob = "*.Dockerfile*" },
```

What's the use of the trailing `*` here? If you're looking to match `*.Dockerfile` then this can be just the extension:

```suggestion
  "Dockerfile",
```

and same for `*.dockerfile*` and the containerfile equivalents below

---

Oh, woops. Forgot to remove that from when I copied from the others. And while we are on that, should I add change `Dockerfile*` to `Dockerfile.*` so that it explicitly needs a dot `Dockerfile.frontend` scheme?